core/src/main/java/de/danoeh/antennapod/core/service/download/handler/FeedSyncTask.java

```diff
@@ -22,6 +24,9 @@ public FeedSyncTask(Context context, DownloadRequest request) {
     }
 
     public boolean run() {
+        final NewEpisodesNotification newEpisodesNotification =
+                request.getFeedfileId() == 0 ? null : new NewEpisodesNotification(request.getFeedfileId());
```

Comment:
Hmm, this adds some complexity here. How about moving the check inside `NewEpisodesNotification`? It can then set `shouldShowNotification` to `false` when the id is 0